support/helpers/spreadsheet_data.rb

```diff
@@ -13,6 +13,10 @@ def create(valid_item_count, invalid_item_count)
     invalid_item_count.times { @data << invalid_mediator.values }
   end
 
+  def practice_data
```

Comment:
Not a biggie but I find this a bit confusing - I think of a practice as address, URL, etc... not just a phone number - maybe we need a better name for the method.